pages/common/git-secret.md

```diff
@@ -0,0 +1,36 @@
+# git secret
+
+> Bash tool which stores private data inside a git repo.
+> More information: <https://github.com/sobolevn/git-secret>.
+
+- Initialize `git-secret` in a local repository:
+
+`git secret init`
+
+- Grant access to your current git user email:
```

Comment:
Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
- Grant access using the current current git user's email:
```

Would `to` or `using` be more accurate here?

Reply:
First of all, it's backed by GPG, yes. Also, "to" sounds more accurate IMO. I'll drop "your" though.